File: Java/Boeing737.java
class Boeing737 extends Pesawat {

    private String varianSeri;
    private double rentangSayapMeter;
    private double kapasitasKargoKg;
    private String tipeMesinTurbofan;

    public Boeing737() {
        super();
    }

    // konstruktor dengan parameter
    public Boeing737(String jenisBahanBakar, int kapasitasPenumpang, int tahunProduksi, String statusMesin,
                      String maskapaiPemilik, int ketinggianMaksimalKaki, String rutePenerbangan, int jumlahMesin,
                      String varianSeri, double rentangSayapMeter, double kapasitasKargoKg, String tipeMesinTurbofan) {
        super(jenisBahanBakar, kapasitasPenumpang, tahunProduksi, statusMesin,
              maskapaiPemilik, ketinggianMaksimalKaki, rutePenerbangan, jumlahMesin);
        this.varianSeri = varianSeri;
        this.rentangSayapMeter = rentangSayapMeter;
        this.kapasitasKargoKg = kapasitasKargoKg;
        this.tipeMesinTurbofan = tipeMesinTurbofan;
    }

    // getter setter
    public String getVarianSeri() {
        return varianSeri;
    }

    public void setVarianSeri(String varianSeri) {
        this.varianSeri = varianSeri;
    }

    public double getRentangSayapMeter() {
        return rentangSayapMeter;
    }

    public void setRentangSayapMeter(double rentangSayapMeter) {
        this.rentangSayapMeter = rentangSayapMeter;
    }

    public double getKapasitasKargoKg() {
        return kapasitasKargoKg;
    }

    public void setKapasitasKargoKg(double kapasitasKargoKg) {
        this.kapasitasKargoKg = kapasitasKargoKg;
    }

    public String getTipeMesinTurbofan() {
        return tipeMesinTurbofan;
    }

    public void setTipeMesinTurbofan(String tipeMesinTurbofan) {
        this.tipeMesinTurbofan = tipeMesinTurbofan;
    }
}

File: Cpp/Boeing737.cpp
#include <iostream>
#include "Pesawat.cpp"
using namespace std;

class Boeing737 : public Pesawat {

    private:
        string varianSeri;
        double rentangSayapMeter;
        double kapasitasKargoKg;
        string tipeMesinTurbofan;

    public:
        Boeing737() {
        }

        // konstruktor dengan parameter
        Boeing737(string jenisBahanBakar, int kapasitasPenumpang, int tahunProduksi, string statusMesin,
                string maskapaiPemilik, int ketinggianMaksimalKaki, string rutePenerbangan, int jumlahMesin,
                string varianSeri, double rentangSayapMeter, double kapasitasKargoKg, string tipeMesinTurbofan)
            : Pesawat(jenisBahanBakar, kapasitasPenumpang, tahunProduksi, statusMesin,
                    maskapaiPemilik, ketinggianMaksimalKaki, rutePenerbangan, jumlahMesin) {
            this->varianSeri = varianSeri;
            this->rentangSayapMeter = rentangSayapMeter;
            this->kapasitasKargoKg = kapasitasKargoKg;
            this->tipeMesinTurbofan = tipeMesinTurbofan;
        }

        // getter setter
        string getVarianSeri() {
            return varianSeri;
        }

        void setVarianSeri(string varianSeri) {
            this->varianSeri = varianSeri;
        }

        double getRentangSayapMeter() {
            return rentangSayapMeter;
        }

        void setRentangSayapMeter(double rentangSayapMeter) {
            this->rentangSayapMeter = rentangSayapMeter;
        }

        double getKapasitasKargoKg() {
            return kapasitasKargoKg;
        }

        void setKapasitasKargoKg(double kapasitasKargoKg) {
            this->kapasitasKargoKg = kapasitasKargoKg;
        }

        string getTipeMesinTurbofan() {
            return tipeMesinTurbofan;
        }

        void setTipeMesinTurbofan(string tipeMesinTurbofan) {
            this->tipeMesinTurbofan = tipeMesinTurbofan;
        }
};